plugins/ibm_qradar/icon_ibm_qradar/util/constants/constant.py

```diff
@@ -0,0 +1,4 @@
+"""Character constants."""
+
+HTTPS = "https"
+SUCCESS_RESPONSE_CODE = [200, 201, 206]
```

Comment:
```suggestion
SUCCESS_RESPONSE_CODES = [200, 201, 206]
```
Since this is a collection, not a singular value